packages/grafana-data/src/transformations/matchers/readOnlyMatcher.ts

```diff
@@ -0,0 +1,33 @@
+import { FieldMatcher } from '../../types';
+import { fieldMatchers } from '../matchers';
+import { FieldMatcherID } from './ids';
+
+export interface ReadOnlyFieldMatcherOptions<T = any> {
+  innerId: FieldMatcherID;
+  innerOptions: T;
+  formattedValue: string;
+}
+
+const defaultMatcher: FieldMatcher = () => false;
+
+fieldMatchers.register({
+  id: FieldMatcherID.readOnly,
```

Comment:
We can exclude the readonly matchers from the matchers select as these are not usable for people 🙈 
![image](https://user-images.githubusercontent.com/2376619/102221097-04439100-3ee2-11eb-9b1b-255fc071c6bb.png)